website_event_questions_template/models/res_config.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+
+
+class EventMailConfigSettings(models.TransientModel):
+    _inherit = 'event.config.settings'
+
+    event_question_template_id = fields.Many2one(
+        comodel_name='event.question.template',
+        string='Questions Template',
```

Comment:
String should be `Default questions template`. The problem I see with this is that if you set a default template, this template will always be put on new events, and thus questions will be filled, so if you want to put another template, you will need to previously empty the question list manually, which is a total pain.

I would directly drop this feature. It's very easy to select a template when creating an event.

Reply:
I'm remembering that this comment doesn't apply as this is loaded on demand, but any way I still think a default one is not the best option (as in v11 we will be able to put this in an onchange - Please add a comment about this in "Known issues / Roadmap").